infrastructure/terraform/functionKeys/main.tf

```diff
@@ -3,16 +3,29 @@
 ##################################################################################
 
 # Get the functions keys out of the app
-resource "azurerm_template_deployment" "function_keys" {
+resource "azurerm_resource_group_template_deployment" "function_keys" {
```

Comment:
Why the name "azurerm_resource_group_template_deployment" instead of "azurerm_template_deployment" ?

Reply:
I migrated to azurerm_resource_group_template_deployment because of a warning that azurerm_template_deployment will be deprecated. See the note on top of
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/template_deployment